data/routes/scripts/standard_ot_api.lua

```diff
@@ -0,0 +1,60 @@
+local json = nil
+
+local function getPlayersOnline(session)
+	session.responder:setResponseStatus(200, 'OK')
+	session.responder:setResponseBody(json.encode(Game.getPlayers()))
+end
+
+local function getServerStatus(session)
+	local data = {
+		uptime = getWorldUpTime(),
+		ip = configManager.getString(configKeys.SERVER_IP),
+		serverName = configManager.getString(configKeys.SERVER_NAME),
+		port = configManager.getNumber(configKeys.LOGIN_PORT),
+		location = configManager.getString(configKeys.LOCATION),
+		url = configManager.getString(configKeys.URL),
+		server = 'The Forgotten Server',
+		version = '1.3',
+		client = '1098',
```

Comment:
I thorougly admire your work but I guess you do suck at grepping 🤣 🤣 🤣 
https://github.com/otland/forgottenserver/blob/master/src/luascript.cpp#L4258-L4266

And I think it would be sweet to have it for the server name and version too.